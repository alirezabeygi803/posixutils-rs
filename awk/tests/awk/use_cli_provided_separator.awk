BEGIN {
	print FS;
}

{
	print $1, $2, $3
}