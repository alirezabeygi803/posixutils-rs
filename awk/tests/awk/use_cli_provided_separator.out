,
1 Jane janitor
2 Smith sailor
3 Roselyn radiologist
4 Megan machinist
5 Celine clergy
6 Fredrick farmer
7 Benjamin programmer
